src/userCodes/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const express = require('express');
+const apiAuth = require('login.dfe.api.auth');
+const config = require('./../config');
+const postCreateCode = require('./postCreateCode');
+const deleteCode = require('./delete');
+const validateCode = require('./validate');
+
+const router = express.Router();
+
+const routeExport = () => {
+  // Add auth middleware.
+  router.use(apiAuth(router, config));
+
+  // Map routed to functions.
+  router.post('/create', postCreateCode);
```

Comment:
Should this be a put as a user can only have 1 code?

Reply:
We didn't state that a user can only have one code... Needs a discussion.